libs/template/src/lib/core/render-aware/render-aware_creator.ts

```diff
@@ -104,40 +108,38 @@ export function createRenderAware<U>(cfg: {
     nextStrategy(nextConfig: string | Observable<string>): void {
       strategyName$.next(nextConfig);
     },
+    rendered$: renderingEffect$,
     activeStrategy$: strategy$,
     subscribe(): Subscription {
-      return new Subscription().add(renderingEffect$.subscribe());
+      return new Subscription().add((renderingEffect$ as ConnectableObservable<U>).connect());
     },
   };
 }
 
+
 function renderWithLatestStrategy<T>(
   strategyChanges$: Observable<RenderStrategy>
 ): MonoTypeOperatorFunction<T> {
```

Comment:
```suggestion
): OperatorFunction<T, RxNotification<T>> {
```

Seems like the return type change a bit with your changes.

Reply:
thx! i will merge it manually, since your suggestion is missing the "import" statement for OperatorFunction